packages/boxel/addon/components/boxel/radio-input/usage.ts

```diff
@@ -2,9 +2,8 @@ import Component from '@glimmer/component';
 import { tracked } from '@glimmer/tracking';
 import { A } from '@ember/array';
 import { action } from '@ember/object';
-import './usage.css';
 
-export default class CustomRadioUsage extends Component {
+export default class RadioInputUsage extends Component {
```

Comment:
I don’t know what it is but something’s broken with `items` in the documentation; it doesn’t show any contents and clicking the button to add causes an error 😳

![image](https://user-images.githubusercontent.com/43280/148100444-d46f182e-ee72-4db6-8cc8-1b6deadd55d5.png)

Reply:
Thanks for noticing, fixing now.